internal/daysteps/daysteps.go

```diff
@@ -12,9 +25,50 @@ const (
 )
 
 func parsePackage(data string) (int, time.Duration, error) {
-	// TODO: реализовать функцию
+	stepsDur := strings.Split(data, ",")
+
+	if len(stepsDur) != 2 {
+		return 0, 0, ErrInvalidLen
+	}
+
+	steps, err := strconv.Atoi(stepsDur[0])
+	if err != nil {
+		return 0, 0, err
```

Comment:
Лучше оборачивать ошибки
```suggestion
		return 0, 0, fmt.Errorf("invalid steps format: %w", err)
```